features/step_definitions/scheduler.rb

```diff
@@ -26,3 +26,25 @@
     raise "Cannot restore scheduler: #{name}" unless @result[:success]
   }
 end
+
+Given /^the #{QUOTED} clusterrole is restored after scenario$/ do |name|
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  patch_json = '[{"op": "remove", "path": "/rules/2/resourceNames/1"}]'
+  _admin = admin
+  teardown_add {
+    opts = {resource: 'clusterrole', resource_name: name, p: patch_json, type: 'json'}
+    @result = _admin.cli_exec(:patch, **opts)
+    raise "Cannot restore clusterrole: #{name}" unless @result[:success]
+  }
+end
+
+Given /^I delete all custom scheduler resources after scenario$/ do
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  teardown_add {
+    step %Q/admin ensures "my-scheduler" deployment is deleted from the "kube-system" project/
+    step %Q/admin ensures "my-scheduler" service_account is deleted from the "kube-system" project/
+    step 'admin ensures "my-scheduler-as-kube-scheduler" clusterrolebinding is deleted'
```

Comment:
Can we have these 3 steps in scenario directly? Or do we keep a convention that custom schedulers in all scenarios will keep that naming?